zilencer/views.py

```diff
@@ -53,6 +53,18 @@ def remote_server_register_push(request, entity, user_id=REQ(),
 
     return json_success()
 
+@has_request_variables
+def remote_server_register_push_notification(request, hostname, contact_email, uuid, api_key):
+
+    remote_server, created = RemoteZulipServer.objects.get_or_create(
+        uuid=uuid,
+        api_key=api_key,
+        hostname=hostname,
+        contact_email=contact_email,
+        last_updated=timezone.now())
+
+    return json_success()
```

Comment:
should return failure if not created, right?

Reply:
@rishig Yes the function should return failure if not created. Added that , Thanks